src/translations/en.json

```diff
@@ -2164,6 +2164,7 @@
             "entities": {
               "name": "Entities",
               "show_header_toggle": "Show Header Toggle?",
+              "state_color": "Show State Color?",
```

Comment:
This as the field label feels weird but at the same time not. I just could think of another wording.

Reply:
`Change Color Automatically?` `Show Auto Color?`

Reply:
Maybe "Use State Color" or "Apply State Color"?
Naming is hard...

Reply:
Color icons based on state

Reply:
```suggestion
              "state_color": "Color Icons based on state",
```